google-cloud-storage/src/main/java/com/google/cloud/storage/Bucket.java

```diff
@@ -118,7 +118,6 @@ public static BucketSourceOption metagenerationNotMatch() {
      * Returns an option for blob's billing user project. This option is only used by the buckets with
      * 'requester_pays' flag.
      */
-    @GcpLaunchStage.Alpha
     public static BucketSourceOption userProject(String userProject) {
```

Comment:
Is the IAM stuff GA now?

Reply:
Yes. GA'd last year. Launch Id: 179764

Reply:
Requester pays as well. Launch Id: 191976